CMakeLists.txt

```diff
@@ -671,4 +683,27 @@ install(
     EXPORT "${TARGETS_EXPORT_NAME}"
     NAMESPACE "${namespace}"
     DESTINATION "${config_install_dir}"
+    COMPONENT libh3-dev
 )
+
+# Debian package build
+set(CPACK_DEB_COMPONENT_INSTALL 1)
+set(CPACK_GENERATOR "DEB")
+set(CPACK_DEBIAN_FILE_NAME "DEB-DEFAULT")
+# set(CPACK_DEBIAN_PACKAGE_MAINTAINER "TEST PACKAGE") # Required
```

Comment:
Why is the package maintainer not set?

Reply:
I wasn't 100% sure how to set this at the moment. I need to go through the Debian packaging documentation to set all the fields. It will probably be set to something like "Isaac Brodsky <debian@...>" or "H3 Open Source Team <h3-team@...>" depending on whether there's a public alias for the H3 maintainers as a whole.